rust/agama-server/src/network/model.rs

```diff
@@ -737,6 +738,7 @@ pub struct InvalidMacAddress(String);
 
 #[derive(Debug, Default, Clone, PartialEq, Serialize, utoipa::ToSchema)]
 pub enum MacAddress {
+    #[schema(value_type = String, format = "MAC address in EUI-48 format")]
```

Comment:
now I am confused. it should have descriptuon from that MacAddr6 schema and also its value type should be read from it, not?

Reply:
It should, but it does not work in that way. To make it work, I would need to convert the variant into a struct-based one:

```
pub enum MacAddress {
  MacAddress {
    address: macaddr::MacAddr6
  },
  Preserve,
  // etc.
}
```

Otherwise, the macro did not work (at least for me).